src/action.ts

```diff
@@ -280,27 +279,26 @@ class Action {
       }
     }
```

Comment:
Lines from 239 to here (281) can be removed, all the content we're downloading there is included within the bundle